addOns/soap/src/test/java/org/zaproxy/zap/extension/soap/Sample.java

```diff
@@ -149,4 +149,27 @@ public static HttpMessage setInvalidFormatResponse(HttpMessage msg) {
                         + "</soapenv:Envelope>");
         return msg;
     }
+
+    public static HttpMessage setSoapVersionTwoRequest(HttpMessage msg)
+            throws HttpMalformedHeaderException {
+        msg.setRequestHeader(
+                "POST https://www.example.com/Soap12Endpoint/ HTTP/1.1"
+                        + "User-Agent: Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.0;)"
+                        + "Pragma: no-cache"
+                        + "Cache-Control: no-cache"
+                        + "Content-Length: 275"
+                        + "Content-Type: application/soap+xml; charset=UTF-8; action= https://www.example.com/xml/sayHelloWorld"
+                        + "Host: www.example.com");
+        msg.setRequestBody(
+                "<?xml version='1.0' encoding= 'UTF-8' ?>"
+                        + "<s12:Envelope xmlns:s12='http://www.w3.org/2003/05/soap-envelope'>"
+                        + "<s12:Body>"
+                        + "<ns:sayHelloWorld xmlns:ns='http://main.soaptest.org'>"
+                        + "<ns:args0>paramValue</ns:args0>"
+                        + "</ns:sayHelloWorld>"
+                        + "</s12:Body>"
+                        + "</s12:Envelope>");
```

Comment:
Should this have a `</xml>`?

Reply:
There's note over [here](https://www.w3schools.com/xml/xml_syntax.asp) that says,
> Note: The XML prolog does not have a closing tag! This is not an error. The prolog is not a part of the XML document.

Reply:
Perfect, thanks!